packages/nodejs/.changesets/add-config-options-for-disabling-default-instrumentation.md

```diff
@@ -0,0 +1,24 @@
+---
+bump: "patch"
+type: "add"
+---
+
+Add config options for disabling default instrumentation like HTTP, HTTPS, PostgreSQL (pg package) and Redis (node-redis package).
+
+The following configuration options have been added:
+
+- `instrumentHttp`
+- `instrumentHttps`
```

Comment:
```suggestion
```

Since the HTTP and HTTPS settings have been merged into `instrumentHttp`, the `instrumentHttps` entry should be removed from the changelog.